vm/src/types/slot.rs

```diff
@@ -43,7 +45,7 @@ pub struct PyTypeSlots {
     // More standard operations (here for binary compatibility)
     pub hash: AtomicCell<Option<HashFunc>>,
     pub call: AtomicCell<Option<GenericMethod>>,
-    // tp_str
+    pub repr: AtomicCell<Option<ReprFunc>>,
```

Comment:
As `repr` use always object `repr` if there is no `tp_repr` slot, do we need to use `AtomicCell<Option< .. >>` here? Is there any other way like using default implemtation and overriding

Reply:
I'm really unsure about that part 😅
I agree that it doesn't make sense for an object not to have a `repr` slot defined but I don't know how to wire the default object slot here.

Reply:
Ah.. let me take-back what I commented before. Im not sure exactly, we need to use AtomicCell<Option<..>> at now for updating slot from python-side code. I'm sorry.

Reply:
Ah no problem, I'm also itching to remove that superfluous option check 🙂